src/jit/gentree.h

```diff
@@ -2429,6 +2524,14 @@ struct GenTreeCall final : public GenTree
     }
 #endif // !LEGACY_BACKEND
 
+    // Returns true if the call has retBuf argument
+    bool HasRetBufArg() const { return (gtCallMoreFlags & GTF_CALL_M_RETBUFFARG) != 0; }
```

Comment:
I agree that it makes sense to have the method on a call node, but it should be implemented using the ReturnTypeDesc, as it is a characteristic of the way the value is returned. We would like to centralize the information about **_how_** values are passed, and this falls into that category.